src/parser/entities/image/types.ts

```diff
@@ -0,0 +1,55 @@
+import type { Point2D, Point3D } from '../../../types';
+import type { CommonDxfEntity } from '../shared';
+import type { ImageClipMode, ImageClippingBoundaryType } from './consts';
+
+export interface ImageEntity extends CommonDxfEntity {
```

Comment:
We don't have any single sample file having IMAGE ENTITY inside, which should be saved from production environment.
So I'm not sure which one is optional and which one is not.
Note that official document is not reliable source of determining optionality. (That's shameful!)